drivers/hmis/app/graphql/types/hmis_schema/enrollment.rb

```diff
@@ -200,6 +201,10 @@ def open_enrollment_summary
       load_ar_association(client, :enrollments).where.not(id: object.id).open_including_wip
     end
 
+    def last_current_living_situation
+      object.current_living_situations.order(information_date: :desc).first
```

Comment:
We should use a data loader here so we're not doing n+1 queries. Something like `load_ar_association(object, :current_living_situations).max_by(:information_date)`

Reply:
Fixed